scalardl/project.clj

```diff
@@ -13,7 +13,7 @@
                    :plugins [[lein-cloverage "1.1.2"]]}
              :use-released {:dependencies [[com.scalar-labs/scalardl-java-client-sdk "4.0.0-SNAPSHOT"
                                             :exclusions [org.slf4j/slf4j-log4j12
-                                                         com.oracle.database.jdbc/ojdbc8-production
+                                                         com.oracle.database.jdbc/ojdbc8
```

Comment:
This is not related to the issue directly, but the latest ScalarDB doesn't depend on `com.oracle.database.jdbc/ojdbc8-production` but `com.oracle.database.jdbc/ojdbc8`.